roles/ssh_hardening/templates/openssh.conf.j2

```diff
@@ -81,6 +81,11 @@ StrictHostKeyChecking ask
 {# This outputs 'KexAlgorithms <list-of-algos>' if ssh_kex is defined or '#KexAlgorithms' if ssh_kex is undefined #}
 {{ 'KexAlgorithms ' ~ ssh_kex|join(',') if ssh_kex else 'KexAlgorithms'|comment }}
 
+# Specifies the host key algorithms that the client wants to use in order of preference.
+{% if sshd_version is version('5.8', '>=') %}
```

Comment:
Are you sure the version comparison here is needed?
In the releasenotes there is a mention of this option since version 2.9: https://www.openssh.com/txt/release-2.9

Reply:
My bad - thanks!
I made the wrong assumption that it was same for the client as for the server. Will fix it.